exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/QueryResources.java

```diff
@@ -83,10 +83,12 @@ public QueryResult submitQueryJSON(QueryWrapper query) throws Exception {
   @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
   @Produces(MediaType.TEXT_HTML)
   public Viewable submitQuery(@FormParam("query") String query,
-                              @FormParam("queryType") String queryType) throws Exception {
+                              @FormParam("queryType") String queryType,
+                              @FormParam("autoLimit") String autoLimit
+                              ) throws Exception {
     try {
       final String trimmedQueryString = CharMatcher.is(';').trimTrailingFrom(query.trim());
-      final QueryResult result = submitQueryJSON(new QueryWrapper(trimmedQueryString, queryType));
+      final QueryResult result = submitQueryJSON(new QueryWrapper(trimmedQueryString, queryType, autoLimit)); //QueryWrapper will validate autoLimit
```

Comment:
I think the comment is redundant and may be removed: 
```suggestion
      final QueryResult result = submitQueryJSON(new QueryWrapper(trimmedQueryString, queryType, autoLimit)); 
```

Reply:
Ok